### DISCUSSION TITLE
Please explain step 4 in details

### DISCUSSION THREAD

**Discussion by chrisnagib** [mtalcott/google-photos-deduper#45] [Q&A] [answered]
I have no experience but I walked through the guide and did all the previous steps 
but I don't understand how and where to setup local environment variables neither the follwoing lines, 

> 4. Set up local environment variables.
> cp example.env .env
> Generate [FLASK_SECRET_KEY](https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY) with python -c 'import secrets; print(secrets.token_hex())' and add it to .env.
> Add GOOGLE_CLIENT_ID and GOOGLE_CLIENT_SECRET from the client_id and client_secret values from the client secret file created above.

please explain 
Thanks

**Comment by chrisnagib** [comment]
any help!!

**Comment by chrisnagib** [comment]
guys I know it may be obvious for you but I really need to do it 

thanks

**Reply by scottv127** [reply]
I'm looking for help with the same; did you find out how to do it yet?  Thanks. Scott

**Reply by zunaidFarouque** [reply]
same here. help needed

**Comment by scottv127** [comment]
I figured it out.  First I created an OpenAI account and then just asked it
the questions and I provided the syntax of what to type into a terminal (on
a Mac).  I also needed to load python which OpenAI can instruct you to do
if you ask.

Send another note if you’re still stuck.  I didn’t write down the steps but
could probably recreate in OpenAI.

**Comment by brianhughesspe** [comment, marked as answer]
I just posted a detailed explainer here as well: [https://github.com/mtalcott/google-photos-deduper/discussions/46](https://github.com/mtalcott/google-photos-deduper/discussions/46)
